api-admin/src/main/java/bio/terra/pearl/api/admin/service/enrollee/EnrolleeExtService.java

```diff
@@ -43,7 +49,12 @@ public List<Enrollee> findForKitManagement(
     return enrolleeService.findForKitManagement(studyShortcode, environmentName);
   }
 
-  public Enrollee findWithAdminLoad(AdminUser operator, String enrolleeShortcodeOrId) {
+  public Enrollee findWithAdminLoad(
```

Comment:
Let's convert this to use @EnforcePortalStudyEnvPermission(perm= "participant_data_view").  Or, if you're ambitious, create a new @EnforceStudyEnvEnrolleePermission annotation that also handles checking the enrollee is in the study environment

Reply:
if it's okay with you, I'm going to defer this for a later PR. I'd rather tackle this entire service all at once and implement the `EnforceStudyEnvEnrolleePermission` annotation that you proposed- and I'd like to keep this PR very small because I want to guarantee that it gets released tomorrow prior to the live-testing of in-person kits later this week